lib/font-assets-helper.js

```diff
@@ -0,0 +1,227 @@
+const path = require('path');
```

Comment:
Maybe just change the file name from `lib/font-assets-helper.js` to `lib/android-font-assets-helper.js`